hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/TargetManagement.java

```diff
@@ -643,4 +586,27 @@ Long countAllTargetsByTargetFilterQueryAndNotInRolloutGroups(List<RolloutGroup>
     @PreAuthorize(SpringEvalExpressions.HAS_AUTH_UPDATE_TARGET + SpringEvalExpressions.HAS_AUTH_OR
             + SpringEvalExpressions.IS_CONTROLLER)
     Target updateTarget(TargetUpdate update);
+
+    /**
+     * Find {@link Target} based on given ID returns found Target without
```

Comment:
Please rework the javadoc for a clearer description what the method does. 

e.g.: Find a {@link Target} based a given ID. The returned target will not contain details (e.g {@link Target#getTags()} and {@link Target#getActions()})